### ISSUE TITLE
if a value is declared but never used its style is stripped or is black

### ISSUE THREAD

**Issue by kinghat** [vangware/dark-plus-material#10]
![image](https://user-images.githubusercontent.com/16073360/52690055-09200e00-2f22-11e9-997b-c2ba4dbcfb85.png)

**Comment by MariusVatasoiu** [comment]
I can confirm this.

**Comment by Kiraub** [comment]
Ran into the same problem today, turns out a change in the version 2 update did this.

Specifically the color configuration `"editorUnnecessaryCode.opacity": "#0000000A"` is what makes the unused variables appear transparent.

As a quickfix until this gets changed override the option manually by extending your `settings.json` with:

```json
"workbench.colorCustomizations": {
    "editorUnnecessaryCode.opacity": "#000000FF"
},
```

Alternatively, you can also set the value to `"#000000A0"` to make unused variables appear semi-transparent.

**Comment by loucyx** [comment]
@MariusVatasoiu @Kiraub @k1nghat noticed this today, fixed it on 2.0.1. Can you confirm if this is fixed?

**Comment by Kiraub** [comment]
> @MariusVatasoiu @Kiraub @k1nghat noticed this today, fixed it on 2.0.1. Can you confirm if this is fixed?

Can confirm. Tested in a js file and unused params are semi-transparent again. Thanks for the quick fix!

**Comment by loucyx** [comment]
Thanks to all of you for creating the issue! ... In a few hours I'll submit a new version that will patch the vast majority of color issues. Closing this one! ❤️
